src/components/faq/calendar/Calendar.js

```diff
@@ -1,18 +1,78 @@
 import React from 'react';
-
 import styled from "styled-components";
+import { CalendarActiveCSS, CalendarInactiveCSS, CalendarTitleCSS, CalendarDesCSS, CalendarTagCSS} from "./CalendarCSS"
 
-import { colors } from "../../colors/index";
-
-// import css later
+import CalendarTagImage from '../../../assets/tag_filled.svg';
 
 // Calendar
-function Calendar() {
+const CalendarComponent = (props) => {
+    let styles = ""
+    // If description is too long, cut it
+    const checkDescription = () => {
+        let modified_calendar_description = props.description
+        if (props.description.length > 64) {
+            modified_calendar_description = props.description.substring(0, 64) + " ..."
+        }
+    
+        return modified_calendar_description;
+    }
+    const calendarStyles = () => {
+        // true
+        if (props.active) {
+            // Active calendar
+            styles += `
+                ${CalendarActiveCSS}
+            `
+        }
+        // false
+        else {
+            // Inactive calendar
+            styles += `
+                ${CalendarInactiveCSS}
+            `
+        }
+        return styles;
+    }
+
+    const CalendarComponent = styled.div `
+        ${calendarStyles}
+    `
+    const CalendarTitle = styled.p `
```

Comment:
since CalendarTitle and CalendarDes are static, i would put it outside the scope of the react component, above line 8. this will result in less react rerenders and make the component more efficient :)